cartridges/app_storefront_base/cartridge/controllers/Default.js

```diff
@@ -26,7 +26,7 @@ server.get(
     "Start",
     consentTracking.consent,
     cache.applyDefaultCache,
-    userLoggedIn.validateLoggedIn,
+    // userLoggedIn.validateLoggedIn,
```

Comment:
Please, avoid committing commented code, unless it is for documentation. You can revert all your changes to app_storefront_base files, since we want it to be the same as in the beginning.

Reply:
Ok, I will fix it